docs/src/getting-started/predicates.md

```diff
@@ -1,15 +1,36 @@
 # Predicates
 
-Predicates, in Sway, are programs that return a Boolean value, and they do not have any side effects (they are pure).
+Predicates, in Sway, are programs that return a Boolean value, and they do not have any side effects (they are pure). Assets can be owned by a predicate address. The predicate address is generated from the compiled byte code and is the same as the `P2SH` address used in Bitcoin. Users can seamlessly send assets to the predicate address as they do for any other address. To spend the predicate funds, the user has to provide the original `byte code` of the predicate together with the `predicate data`. The `predicate data` will be used when executing the `byte code`, and if the predicate is validated successfully, the funds can be transferred.
```

Comment:
Since we're touching this, rephrasing some parts from passive to active voice for clarity.

```suggestion
Predicates, in Sway, are programs that return a Boolean value and do not have any side effects (they are pure). A predicate address can own assets. The predicate address is generated from the compiled byte code and is the same as the `P2SH` address used in Bitcoin. Users can seamlessly send assets to the predicate address as they do for any other address. To spend the predicate funds, the user has to provide the original `byte code` of the predicate together with the `predicate data`. The `predicate data` will be used when executing the `byte code`, and the funds can be transferred if the predicate is validated successfully.
```